lib/queue.js

```diff
@@ -884,6 +884,11 @@ Queue.prototype.retryJob = function(job) {
 };
 
 Queue.prototype.toKey = function(queueType){
+  return [this.keyPrefix ? '' : 'bull', this.name, queueType].join(':');
+};
+
+Queue.prototype.toPrefixedKey = function(queueType){
+  if (!this.keyPrefix) return this.toKey(queueType);
   return [this.keyPrefix, this.name, queueType].join(':');
 };
 
```

Comment:
Aha yeah I see what you mean. I suppose that could also work. Maybe having it under redisOptions is a bit misleading though, the api also allows people to pass their own createClient functions but its not clear that they should ignore this flag?